### PR TITLE
More copy edits

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

File: sections/4-development.tex [hunk 1]
```diff
@@ -5,30 +5,32 @@ \section{Development Model}
...
     \item Contributed code must provide unit tests which cover the majority of the new functionality.
-    \item All code must be within scope, and it must be reviewed and accepted by at least two members of the developer community before it is merged into the code base\footnote{This requirement is relaxed for bug fixes or small documentation changes.}.
+    \item All code must be within scope, and must be approved by at least two members of the developer community before it is merged into the code base\footnote{This requirement is relaxed for bug fixes or small documentation changes.}.
 \end{enumerate}
 These requirements are imposed on every pull request regardless of contributor.
 
-As of version 1.0, \sunpypkg consists of 48,427 lines of code\footnote{This number includes documentation and comments.
-There are 30,906 lines of pure code.} contributed by 123 unique contributors over 11,659 \git commits.
+As of Version 1.0, \sunpypkg consists of 48,427 lines of code\footnote{This number includes documentation and comments.
+There are 30,906 lines of code if documentation and comments are not counted.} contributed by 123 unique contributors over 11,659 commits.
```
Comment (hayesla) [inline sections/4-development.tex]: I think a word might be missing here ' As of Version 1.0, \sunpypkg consists of 48,427 lines of code contributed by 123 unique contributors _with_ over 11,958 commits

Maybe I'm reading it wrong though?

File: sections/4-development.tex [hunk 2]
```diff
@@ -5,30 +5,32 @@ \section{Development Model}
...
 The leftmost panel of \autoref{fig:metafig} shows the steady growth of the code base since 2011 with, on average, approximately 16 lines added per day.
-A reduction in the number of lines of code occurred after the release of version 0.9 due to a major clean up effort that led to the deletion of unused code and the removal of support for Python 2.
+A large reduction in the code base occurred after the release of 0.9 due, primarily to, the deletion of unused code and the removal of Python 2 support.
```
Comment (hayesla) [inline sections/4-development.tex]: primarily to--> primarily from?

File: sections/4-development.tex [hunk 3]
```diff
@@ -5,30 +5,32 @@ \section{Development Model}
...
 
 The middle panel of \autoref{fig:metafig} shows the steady rise in the number of unique contributors with, on average, more than one new contributor added each month.
-Additionally, the rightmost panel of \autoref{fig:metafig} shows the distribution of total number of commits per contributor as of June 2019.
+The total number of committers is large for a package of this size in the heliophysics community \citep{pyhc2018}.
+
+The rightmost panel of \autoref{fig:metafig} shows the distribution of total number of commits per contributor as of June 2019.
```
Comment (hayesla) [inline sections/4-development.tex]: of the?

File: sections/5-modules.tex [hunk 4]
```diff
@@ -93,33 +93,33 @@ \subsubsection{\Timeseries}
...
-Other source-specific metadata is also used by the \Map class to specify color tables and appropriate image scaling.
-It is also possible to create a custom \Map from both a 2D data array and the appropriate metadata, such as coordinate information.
+A \Map can be created from local files or using a url to a remote data file.
+The \Map class will automatically detect the instrument and parse the metadata to infer the coordinate system from the appropriate FITS keywords \citep{refId0, 2006A&A...449..791T}.
+Other source-specific metadata is used to determine the appropriate color table and image normalization for visualization.
+It is also possible to create a custom \Map by providing a 2D data array and metadata.
 
-Similar to \Timeseries, the \Map class contains the image data array, coordinate information and relevant metadata as attributes, all of which can be easily accessed by the user.
+Similar to \Timeseries, the \Map class contains the image data array, coordinate information and relevant metadata as attributes.
```
Comment (hayesla) [inline sections/5-modules.tex]: ,

**Review by hayesla — APPROVED** [review]

**Comment by ehsteve** [comment]
thanks @hayesla for the comments!

**Pull request merged**